images/installer/Dockerfile.openstack.ci

```diff
@@ -0,0 +1,27 @@
+# This Dockerfile is a used by CI to test using OpenShift Installer against an OpenStack cloud.
```

Comment:
Can this go in `images/openstack/Dockerfile.ci` or similar with an `OWNERS` file?  The UPI file is here because we expect it to be cross-platform.

Reply:
yep that makes sense. thanks for the review